schema/show.js

```diff
@@ -70,7 +70,8 @@ const ShowType = new GraphQLObjectType({
     name: {
       type: GraphQLString,
       description: 'The exhibition title',
-      resolve: ({ name }) => name.trim(),
+      resolve: ({ name }) =>
+        isExisty(name) ? name.trim() : '',
```

Comment:
Good q - I wasn't sure if clients are displaying it and being `null` might be worse (than a blank string)? I also didn't think it was necessary, i can undo.